eng/pipelines/templates/jobs/ci.yml

```diff
@@ -307,6 +307,9 @@ jobs:
             Selection: sparse
             GenerateVMJobs: true
       PreGenerationSteps:
+        - task: UsePythonVersion@0
```

Comment:
was this intended change?

Reply:
yes to fix pipeline issues, Scott from SDK team referred to specific commit